package.json

```diff
@@ -653,19 +653,31 @@
   "scripts": {
     "start": "stripes serve",
     "lint": "eslint .",
-    "test": "stripes test karma"
+    "test": "yarn run test:unit && yarn run test:e2e",
+    "test:unit": "jest --ci --coverage",
+    "test:e2e": "stripes test karma --karma.singleRun --karma.browsers ChromeDocker --karma.reporters mocha junit --coverage"
   },
   "devDependencies": {
+    "@babel/core": "^7.8.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-decorators": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.7.4",
     "@bigtest/interactor": "^0.7.0",
     "@bigtest/mocha": "^0.5.1",
     "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^5.0.0",
     "@folio/stripes": "^5.0.0",
+    "@folio/stripes-acq-components": "^2.2.0",
```

Comment:
I think more people should have reviewed this PR. I don't think ui-inventory should know about this package. This will make the development under the platform inconvenient. As a part of workspace all deps are popped up and thus running tests will result in errors in order to resolve which the deps should be setup outside the platform.

Reply:
o_O Could you specify any implication or how it can make 

> This will make the development under the platform inconvenient
? I can't see any.
The rest is also unclear to me, what errors can it give?

Reply:
I have this kind of failed tests when I install deps as a part of the platform. In order for this kind of errors to go away, I need to move the ui-inventory out of the platform and install all deps using yarn just for this only project so all deps ends up in the ui-inventory/node_modules as oppose to stripes/node_modules when we do it as a part of the platform. If you don't have such issues, could you please clarify in what way it solves for you? 
![Uploading Screen Shot 2020-11-26 at 12.41.32 PM.png…]()

Reply:
I don't develop module as a platform, it fails for me. I don't have such issues since tests - is a development thing, if you look at devDependencies - you'll find jest there. As well as stripes-acq-components dep.